doc/api/path.md

```diff
@@ -241,82 +333,89 @@ path.parse('/home/user/dir/file.txt')
 // }
 ```
 
-An example on Windows:
+```text
+┌─────────────────────┬────────────┐
+│          dir        │    base    │
+├──────┬              ├──────┬─────┤
+│ root │              │ name │ ext │
+"  /    home/user/dir / file  .txt "
+└──────┴──────────────┴──────┴─────┘
+(all spaces in the "" line should be ignored -- they're purely for formatting)
+```
+
+On Windows:
 
 ```js
-path.parse('C:\\path\\dir\\index.html')
+path.parse('C:\\path\\dir\\file.txt')
 // returns
 // {
 //    root : "C:\\",
 //    dir : "C:\\path\\dir",
-//    base : "index.html",
-//    ext : ".html",
-//    name : "index"
+//    base : "file.txt",
+//    ext : ".txt",
+//    name : "file"
 // }
 ```
 
+```text
+┌─────────────────────┬────────────┐
+│          dir        │    base    │
+├──────┬              ├──────┬─────┤
+│ root │              │ name │ ext │
+" C:\      path\dir   \ file  .txt "
+└──────┴──────────────┴──────┴─────┘
+(all spaces in the "" line should be ignored -- they're purely for formatting)
+```
+
+A [`TypeError`][] is thrown if `path` is not a string.
+
 ## path.posix
 
-Provide access to aforementioned `path` methods but always interact in a posix
-compatible way.
+The `path.posix' property provides access to POSIX specific implementations
+of the `path` methods.
 
 ## path.relative(from, to)
 
-Solve the relative path from `from` to `to`. `from` and `to` must be strings.
+* `from` {String} A file path
+* `to` {String} A file path
 
-At times we have two absolute paths, and we need to derive the relative
-path from one to the other.  This is actually the reverse transform of
-`path.resolve`, which means we see that:
+The `path.relative()` method returns the relative path from `from` to `to`.
+If `from` and `to` are equal, a zero-length string is returned.
```

Comment:
If we're going to mention equality here, we should probably include a note that also says that both `from` and `to` are `path.resolve()`d first before doing anything else (including checking for equality).